source/features/clean-image-auto-links.tsx

```diff
@@ -0,0 +1,31 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import GitHubURL from '../github-helpers/github-url';
+
+function init(): void {
+	for (const image of select.all('.markdown-body a > img')) {
+		const link = image.parentElement as HTMLAnchorElement;
+
+		if (image.src === link.href) {
+			link.replaceWith(image);
+		}
+
+		if (pageDetect.isSingleFile(link)) {
+			const rawUrl = new GitHubURL(link.href).assign({route: 'raw'}).toString();
```

Comment:
does `href` work here?

Reply:
It works fine? Not sure what are you confirming.

Reply:
instead of tostring()

Reply:
I see, they are identical so it should work.